src/App.tsx

```diff
@@ -4,7 +4,7 @@ import { Routes } from './routes/Routes';
 import './styles/Globals.scss';
 
 const App: React.FC = () => (
-  <Router>
+  <Router basename={process.env.PUBLIC_URL}>
```

Comment:
why it was added?

Reply:
This attribute is necessary for routing to work properly.

Reply:
kk